source/index.html.md

```diff
@@ -93,6 +93,15 @@ easy rolling of API keys. A merchant account usually uses one API key.
 API keys can have an expire date. API keys can be enabled or disabled. For
 merchant accounts, these details can be observed in the Clearhaus Dashboard.
 
+An API key points to an account. An account can have multiple API keys. For
```

Comment:
Very similar to line 89.

Reply:
Resolved by 9e24cc12c69ca2c7a5118266cbd5c8512baa1fe4